guides/moving-a-repo-to-github.md

```diff
@@ -0,0 +1,64 @@
+# Transferring a repo from git.dxw.net to github
+
+To allow better access to our code, for us to do code reviews, and run tests, we’re now switching most of our repos to be on Github. This guide will take you through a step-by-step of how to set up a new repo on Github and what to do with the deployment repo on GovPress.
+
+For more information on deploying using the new repository setup, see the [deploying guide](https://github.com/dxw/playbook/blob/master/guides/deploying.md).
+
+## Create the new repo
+
+Create a new private repository on Github: [https://github.com/organizations/dxw/repositories/new](https://github.com/organizations/dxw/repositories/new)
+
+The naming convention to follow is: `{organisation}-{project}-{repo}`. (repo = “app” or “etc” or “tools”)
+
+i.e. judiciary-intranet-app
+
+If it’s a shared theme, `{organisation}-theme-{theme name}`.
+
+i.e. judiciary-theme-judiciary
+
+## Copy across from old repository
+
+From the old repository, set up the new Github repo with a copy of the current branches and tags:
+
+	# go to your local copy of the old repo
+	cd the-old-repo
+	# fetch
+	git fetch
+	# Create a new master branch as a copy of the current production branch (don’t do this for themes)
+	git branch -f master origin/production
+	# push all local branches
+	git branch | cut -b 3- | xargs git push origin 
+	# duplicate all branches on the origin to the new repo (and duplicate local tags)
+	git push git@github.com:dxw/judiciary-mainsite-app.git refs/remotes/origin/*:refs/heads/* refs/tags/*:refs/tags/*
```

Comment:
I can totally see myself copying this line, pasting it, and hitting enter without remembering to replace "judiciary-mainsite-app". Can we `s/judiciary-mainsite-app/name-of-repo/` so that nobody accidentally fills a real repo up with garbage?